src/config/jest.ts

```diff
@@ -0,0 +1,9 @@
+import redis from '../config/redis';
+
+global.beforeEach(() => {
+  redis.flushall();
```

Comment:
if from the syntax of the redis config, this is made async, right?

Reply:
oh correct, we need to wait for this one. But for the afterAll operation, I don't think we need to await it, right?

Reply:
or choose only one, want to use after or before ?

Reply:
> or choose only one, want to use after or before ?

what do you think about it 😁@

Reply:
> If we only use beforeEach, then after all the tests run, some of the caches will still remain in dev environment.
> If we only use afterEach, then the first test may still contain caches from dev.
> That's why I think we still need to use both of them, but the await should only be used in beforeEach only.

I agree with you, thank you